src/pkg/oci/utils.go

```diff
@@ -91,3 +94,36 @@ func RemoveDuplicateDescriptors(descriptors []ocispec.Descriptor) []ocispec.Desc
 	}
 	return list
 }
+
+// GetInitPackageURL returns the URL for the init package for the given architecture and version.
+func GetInitPackageURL(arch, version string) string {
+	return fmt.Sprintf("ghcr.io/defenseunicorns/packages/init:%s-%s", arch, version)
+}
+
+// DownloadPackage downloads the given OCI package and saves as a tarball.
+func DownloadPackage(url, destinationTarball string, concurrency int) error {
```

Comment:
```suggestion
func PullTarball(url, destinationTarball string, concurrency int) error {
```
Would this make sense as `PullTarball` and then being moved to `pull.go` that way it is more discoverable to library users who may want it?

Reply:
Functions related to OCI / OrasRemote that are not receiver methods on `OrasRemote` are contained in `utils.go`. Pull operations usually have hook-ins related to the way layers are treated. This function is very "dumb", which is why it is called `DownloadPackage`, and does not re-use verbiage contained elsewhere.

Reply:
Ok that makes sense - can we call the function `DownloadPackageTarball`` or `DownloadTarball`? Just to de-conflict with `PullPackage` since that pulls package layers into a dir which is a bit different than this.

Reply:
sure